### PR TITLE
[WIP]Added text to speech instructions

### PR DESCRIPTION
**Pull request by vachan-maker**
https://github.com/organicmaps/organicmaps/issues/2383

Still working on it.

### CODE REVIEW

**Comment by vachan-maker** [comment]
Hmm..I forgot to add signed off

**Comment by biodranik** [comment]
Thanks! Do you have other devices to take more different screenshots for different Android versions?
@pastk can you help?

**Comment by biodranik** [comment]
It makes sense to provide instructions for different versions of Android, and also some hints on where users can install a proper TTS engine. Some hints can be extracted from existing TTS issues: https://github.com/organicmaps/organicmaps/issues?q=is%3Aissue+is%3Aopen+tts+

**Comment by vachan-maker** [comment]
No, I don't have any other devices.  Yes, I will add the instructions from where users can add a tts engine.

**Comment by pastk** [comment]
> Thanks! Do you have other devices to take more different screenshots for different Android versions? @pastk can you help?

I have a custom a9.0 and a5.0 emulator only.

I doubt its reasonable to include screenshots for many versions of vendorized UIs. Probably its better to describe (and illustrate with screenshots) a single universal way of searching for "Text-to-speech" settings.

**Comment by vachan-maker** [comment]
These instructions are from Google

https://support.google.com/accessibility/android/answer/6006983?hl=en

They have only listed one method

**Comment by vachan-maker** [comment]
Added instructions for text to speech on Lineage OS

File: content/text-to-speech.md [hunk 1]
```diff
@@ -35,10 +35,9 @@ If you use custom rom like Lineageos without Google Play services & Speech Servi
 1) Install [eSpeak TTS engine](https://f-droid.org/en/packages/com.reecedunn.espeak) available on F-Droid
 
 2) Set it as the preferred engine
-a) Go to Lineageos main **Settings**
-b) Scroll down to **Accessibility**
-c) Tap on **text-to-speech output** (below SCREEN READERS)
-d) tap on **Preferred engine** (left side) and make sure **eSpeak** is selected
+
+      Go to Lineage os main **Settings** --> Scroll down to **Accessibility** --> Select **text-to-speech output** (below SCREEN READERS)--> **Preferred engine** (left side) and make sure **eSpeak** is selected
```
Comment (biodranik) [inline content/text-to-speech.md:39]: This is a markdown, you can make a numbered list.

Comment (vachan-maker) [reply content/text-to-speech.md]: Is this enough?

**Review by rtsisyk — APPROVED** [review]
LGTM

File: content/text-to-speech.md (hunk 1 above)
Comment (foss-) [reply content/text-to-speech.md]: It's `LineageOS`, see https://lineageos.org/ or https://github.com/lineageos

**Comment by rtsisyk** [comment]
Hi @vachan-maker , sorry for not getting back to you sooner on this PR. Let's merge it and move forward. I need two things:

1. Fix `zola serve`

```
Error: Reason: Variable `alt` not found in context while rendering 'shortcodes/screenshot.html'
```

2. Add Signed-off-by (git commit --amend -s)

https://github.com/organicmaps/organicmaps/blob/master/docs/CONTRIBUTING.md#submitting-your-changes

I promise that we will move forward faster with reviews. This repo just wasn't on my radar.
The further plan is to integrate FAQ from the app (in main repo) and rework the build process to get the generated version from website when building the app.

**Comment by vachan-maker** [comment]
Hi  I am really sorry for not completing this.
Is this PR still required? Or should I close it?

**Comment by meenbeese** [comment]
> Hi  I am really sorry for not completing this.
> Is this PR still required? Or should I close it?

We appreciate your initial work and are close to finishing it in #171 and #185. Therefore, it makes sense to close this PR in favor of those, where you can review the work and contribute your ideas instead. @vachan-maker

**Pull request closed without merging**